app/controllers/manage_tools_controller.rb

```diff
@@ -0,0 +1,13 @@
+# :nodoc:
+class ManageToolsController < ManageMetadataController
+  before_action :umm_t_enabled?
+
+  def show
+    # If you change this number you must also change it in the corresponding test file - features/manage_tools/open_drafts_spec.rb.
+    @draft_display_max_count = 5
+
+    @drafts = ToolDraft.where(provider_id: current_user.provider_id)
```

Comment:
Not important to make a change on it's own, but if other things are changed: is the alignment here correct in the file? It doesn't seem to be in the web diff